pkgs/top-level/by-name-overlay.nix

```diff
@@ -32,27 +36,35 @@ let
       # Additionally in either of those alternatives, we would have to duplicate the hardcoding of "README.md"
       { }
     else
-      mapAttrs (name: _: baseDirectory + "/${shard}/${name}/package.nix") (
+      mapAttrs (name: _: import (baseDirectory + "/${shard}/${name}/package.nix")) (
```

Comment:
I fear touching this may affect `meta.position`, `meta.maintainersPosition` or `meta.teamsPosition`. Could you check if that is the case?